src/components/IOUConfirmationList.js

```diff
@@ -101,6 +102,45 @@ const defaultProps = {
 const MINIMUM_BOTTOM_OFFSET = 240;
 
 class IOUConfirmationList extends Component {
+    constructor(props) {
+        super(props);
+
+        this.toggleOption = this.toggleOption.bind(this);
+
+        const formattedSelectedParticipants = this.getFormattedSelectedParticipants(this.props.participants);
+
+        this.state = {
+            selectedParticipants: formattedSelectedParticipants,
+            unselectedParticipants: [],
+        };
+    }
+
+    /**
+     * Returns the selectedParticipants with amount
+     *
+     * @param {Array} selectedParticipants
+     * @returns {Array}
+     */
+    getFormattedSelectedParticipants(selectedParticipants) {
+        return getIOUConfirmationOptionsFromParticipants(
+            selectedParticipants,
+            this.props.numberFormat(this.calculateAmount(selectedParticipants) / 100, {
+                style: 'currency',
+                currency: this.props.selectedCurrency.currencyCode,
+            }),
+        );
+    }
+
+    /**
+     * Returns the unselectedParticipants without amount
+     *
```

Comment:
unnecessary new line

Reply:
I'm not sure about this. Which line is unnecessary? I formatted it based on methods in IOUModal
https://github.com/Expensify/Expensify.cash/blob/c22f7a9db7eb796612d71a48af5457a634a77f45/src/pages/iou/IOUModal.js#L157-L162